### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/silverstripe-advancedworkflow/actions?query=branch%3Apulls%2F5.6%2Fmodule-standards

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+6
```
Comment (dhensby) [inline .nvmrc]: Is this addition intentional?

Comment (emteknetnz) [reply .nvmrc]: Yup, new CI wants an `.nvmrc` file defined - https://github.com/creative-commoners/silverstripe-advancedworkflow/runs/7194338426?check_suite_focus=true

I don't actually don't know what version it's supposed to be some I'm just guessing :-)

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Please update badge in readme

**Comment by emteknetnz** [comment]
Updated

**Review by GuySartorelli — APPROVED** [review]
